src/common/catalog/src/consts.rs

```diff
@@ -27,3 +27,4 @@ pub const SYSTEM_CATALOG_TABLE_ID: u32 = 0;
 pub const SCRIPTS_TABLE_ID: u32 = 1;
 
 pub const MITO_ENGINE: &str = "mito";
+pub const IMMUTABLE_FILE_ENGINE: &str = "immutable-file";
```

Comment:
suggestion(non-blocking): The engine name is too verbose. I'd prefer just `file`.